### ISSUE TITLE
Custom Potions

### ISSUE THREAD

**Issue by GetDerpy** [cyberpwnn/GlacialRealms#250]
**Up for discussion**

Custom potions would be obtained from an NCP at spawn. https://wiki.citizensnpcs.co/Alchemist

Suggested Potions

**Wizards Escape**
Recipe: 20 Emerald Blocks, 64 sugar, glass bottle
Effect: Speed 4
Duration: 15 Seconds

**Blood of the Dragon**
Recipe: 4 Emerald Blocks, 64 Magma Cream, glass bottle
Effect: Fire Resistance
Duration: 20 Minutes

**Poseidons Blessing**
Recipe: 4 Emerald Blocks, 64 lilly pad, glass bottle
Effect: Water breathing
Duration: 20 Minutes

**Miners Spirit**
Recipe: 8 Emerald Blocks, 64 golden carrot, glass bottle
Effect: Haste 4
Duration: 10 minutes

**Tears of the Gods**
Recipe: 120 Emerald Blocks, 1 God Apple, glass bottle
Effect: Regeneration 5
Duration: 30 seconds

More possible.

**Comment by cyberpwnn** [comment]
@SwiftSwamp open for discussion

**Comment by SwiftSwamp** [comment]
Sure, we could start work on the Brewing stuff we spoke about a while back, although I still think we should wait a few before starting.

**Comment by SwiftSwamp** [comment]
EDIT: Derpy's suggestion was linking us to a Citizens addon that already does this. Continuing with discussion.

**Comment by cyberpwnn** [comment]
I like brewing better. Its less "hacky". Same goes for crafting.

**Comment by cyberpwnn** [comment]
Really on hold now
